src/es/components/molecules/multiLevelNavigation/MultiLevelNavigation.js

```diff
@@ -190,6 +190,19 @@ export default class MultiLevelNavigation extends Mutation() {
   }
 
   connectedCallback () {
+    const preventDefaultInputSearch = this.root.querySelectorAll(".preventDefaultInputSearch");
+    if (preventDefaultInputSearch.length > 0) {
+      function noScroll(event) {
+        const body = document.body; 
+         if (event.type === 'blur') {
```

Comment:
since your noScroll function gets applied to the 'blur' event, you don't need to check the event.type